spec/views/holds/new.html.erb_spec.rb

```diff
@@ -3,17 +3,43 @@
 require 'rails_helper'
 
 RSpec.describe 'holds/new.html.erb', type: :view do
-  let(:bib) { build(:bib_with_holdables) }
+  let(:form_params) { {
+    catkey: '1',
+    title: 'How to Eat More Pizza',
+    author: 'Samantha Smith',
+    volumetric_calls: [],
+    barcode: '2'
+  } }
 
-  context 'with a new hold with holdables to choose' do
+  context 'without volumetric holdables to choose' do
     before do
-      assign(:bib, bib)
+      assign(:place_hold_form_params, form_params)
     end
 
-    it 'renders an input with a unique id attribute by barcode' do
+    it 'does not render checkbox inputs when there' do
```

Comment:
```suggestion
    it 'does not render checkbox inputs' do
```

Reply:
thanks, I should've read over the plain english specs again I guess! 🙃